centreon/src/Core/TimePeriod/Application/UseCase/AddTimePeriod/DtoException.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types = 1);
+
+namespace Core\TimePeriod\Application\UseCase\AddTimePeriod;
+
+use Symfony\Component\Serializer\Attribute\SerializedName;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class DtoException
+{
+    public function __construct(
+        #[Assert\NotBlank]
+        #[SerializedName('day_range')]
```

Comment:
Have a look on previous PR by Jeremy to enable the case converter.
[Related documentation](https://symfony.com/doc/current/components/serializer.html#camelcase-to-snake-case)